Source/FieldSolver/SpectralSolver/SpectralFieldData.cpp

```diff
@@ -54,6 +54,37 @@ SpectralFieldData::SpectralFieldData( const BoxArray& realspace_ba,
         IntVect fft_size = realspace_ba[mfi].length();
 #ifdef AMREX_USE_GPU
         // Add cuFFT-specific code
```

Comment:
Same comment: please remove `// Add cuFFT-specific code`

You could replace this with `// Create cuFFT plans` (there is a similar comment for the FFTW)

Reply:
Done.